.github/workflows/spark-rapids-jni.yaml

```
@@ -7,7 +7,7 @@ jobs:
   spark-rapids-jni-build:
     runs-on: linux-amd64-cpu8
     container:
-      image: rapidsai/ci-spark-rapids-jni:rockylinux8-cuda12.8.0
+      image: rapidsai/ci-spark-rapids-jni:rockylinux8-cuda12.9.0
```

@pxLi have bumped the Spark RAPIDS CI image to CUDA 12.9.0 above

Is there anything else we need to do?

---

Changing back as discussed here: https://github.com/NVIDIA/spark-rapids/issues/12679#issuecomment-2881910895
```suggestion
      image: rapidsai/ci-spark-rapids-jni:rockylinux8-cuda12.8.0
```